### PR TITLE
refactor(mc-scripts): login command to use new endpoint

### PR DESCRIPTION
**Pull request by emmenko**
Follow up of #2506

### CODE REVIEW

File: packages/mc-scripts/src/utils/auth.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  contactEmail: 'support@commercetools.com',
+});
+
+const getAuthToken = async (mcApiUrl, payload) => {
+  const response = await fetch(`${mcApiUrl}/tokens/cli`, {
+    method: 'POST',
+    headers: {
+      Accept: 'application/json',
+      'Content-Type': 'application/json',
+      'x-user-agent': userAgent,
```
Comment (Rhotimee) [inline packages/mc-scripts/src/utils/auth.js:21]: I think we will need to add `x-mc-cli-access-token` to the headers.

File: packages/mc-scripts/src/utils/auth.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
+const fetch = require('node-fetch');
+const createHttpUserAgent = require('@commercetools/http-user-agent');
+const pkgJson = require('../../package.json');
+
+const calculateExpirationTime = (expiresIn) => Date.now() + expiresIn * 1000;
```
[the remark this answers is not in the document]
Comment (emmenko) [reply packages/mc-scripts/src/utils/auth.js:5]: Yeah I should remove this.

File: packages/mc-scripts/src/utils/auth.js (hunk 1 above)
Comment (Rhotimee) [reply packages/mc-scripts/src/utils/auth.js:21]: Ooh. true.  👍🏽

**Comment by emmenko** [comment]
This is ready to review.

**Review by CarlosCortizasCT — APPROVED** [review]
lgtm

**Review by Rhotimee — APPROVED** [review]

**Pull request merged**